### DISCUSSION TITLE
issue with offset

### DISCUSSION THREAD

**Discussion by ncblx** [jneilliii/OctoPrint-BedLevelVisualizer#477] [General]
Hi,

I have issue with mesh leveling

my setup:
Ender 3 Prod
SKR Mini E3 v2.0
BL touch

How can I change Z to 0? offset ~0.950

Issue:
![136224353-77d0f809-f85b-4a9f-b205-74e87edcbda9 1](https://user-images.githubusercontent.com/27479258/136294913-8f8c302c-69d6-4170-95a9-819f50e9d19c.png)

**Comment by jneilliii** [comment]
try enabling the relative z offset option and press the update mesh now button.

**Reply by ncblx** [reply]
Thank you very much. Issue has been solved.
